schema/07-coingecko.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE token_values(
+    denom TEXT,
+    current_price NUMERIC,
+    market_cap NUMERIC,
+    height INT
```

Comment:
I think it's better to use a `date` column instead of a `height` one. Market prices varies based on time, not block height.